backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/ITUserService.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.cloud.notifications.recipients.itservice;
+
+import com.redhat.cloud.notifications.recipients.itservice.pojo.request.ITUserRequest;
+import com.redhat.cloud.notifications.recipients.itservice.pojo.response.ITUserResponse;
+import io.smallrye.mutiny.Uni;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@RegisterRestClient(configKey = "it-s2s")
+public interface ITUserService {
+
+    @POST
+    @Path("/findUsers")
+    @Produces(MediaType.APPLICATION_JSON)
+    Uni<List<ITUserResponse>> getUsers(ITUserRequest string);
```

Comment:
Shouldn't this be annotated with `@Consumes(APPLICATION_JSON)`?

Reply:
It's working with and without Procudes and Consumes, but I can add it f you want.

Reply:
Probably because `APPLICATION_JSON` is the default in Quarkus right now. It would be safer to have an explicit value though, that kind of default can changed (happened in Quarkus already in the past).

Reply:
let's just go with the annotation to be safe. It doesn't add much complexity so adding a more or less "obsolete" annotation doesn't change much.